data-src/functions.sh

```diff
@@ -0,0 +1,163 @@
+# ensure all primary (chr1~22, X, Y, M) chromosome features float to the top of the file
```

Comment:
is this necessary for public GK?

Reply:
I was thinking I'd rather have the public use the same files as us in case some issue comes up. I'm ok with removing it if you think it's better.

Reply:
I was also thinking we'd change our internal CI to use these scripts, and then it does become important.